packages/core/lib/debug/printer.js

```diff
@@ -209,11 +210,43 @@ class DebugPrinter {
       this.config.logger.log(DebugUtils.formatStack(step.stack));
       this.config.logger.log("");
     }
-    this.config.logger.log(
-      DebugUtils.formatInstruction(traceIndex + 1, totalSteps, instruction)
-    );
-    this.config.logger.log(DebugUtils.formatPC(step.pc));
+
+    this.config.logger.log("Instructions:");
+    this.config.logger.log("");
+
+    // printout instructions
+    // add an ellipse if there exist additional instructions before
```

Comment:
This comment should be moved to before the conditional that displays the eclipse

Reply:
good catch, thx